packages/react/src/primitives/thread/useThreadViewportIsAtBottom.tsx

```diff
@@ -0,0 +1,55 @@
+"use client";
```

Comment:
audit with AI: is this code equivalent to previous one? was anything important changed?